pandas/core/base.py

```diff
@@ -7,8 +7,10 @@
 from pandas.core import common as com
 import pandas.core.nanops as nanops
 import pandas.lib as lib
+from pandas.compat.func_defaults import TRANSPOSE_DEFAULTS
```

Comment:
the entire idea is to make this a 1-liner in the main function call. you can do all kinds of gymnastics in the validation calls, but it is just distracting in the main code body.